xpdacq/beamtime.py

```diff
@@ -637,6 +642,36 @@ def list_bkg(self):
                                           if sa_name.startswith('bkgd')]
         print('\n'.join(contents))
 
+    def robot_location_number(self):
+        print('Please input the location of each sample in the robot'
+              'magazine. If the sample is not in the magazine just leave it '
+              'blank and hit <enter>.')
+        for i, sample in enumerate(self.samples.keys()):
+            print(i, sample)
+            ip = input()
```

Comment:
can we have a more verbose print?

``if Not loaded, simply hit enter and skip`` or something

Reply:
already addressed above. Please ignore this review.